src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.model;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String name, Color color, double radius) {
+        super(name, color);
+        this.radius = radius;
+    }
+
+    public double getArea() {
+        return 3.14 * radius * radius;
+    }
```

Comment:
please create interface AreaCalculator with method getArea()
and make class Figure implement this interface